src/scatter_buildblock/ScatterEstimation.cxx

```diff
@@ -185,6 +187,62 @@ ScatterEstimation(const std::string& parameter_filename)
     }
 }
 
+
+shared_ptr<ProjData> 
+ScatterEstimation::
+make_2D_projdata(const shared_ptr<ProjData> in_3d_sptr)
+{
+    shared_ptr<ProjDataInfo> out_info_2d_sptr(in_3d_sptr->get_proj_data_info_sptr()->create_shared_clone());
+    shared_ptr<ProjData> out_2d_sptr;
+    if (in_3d_sptr->get_proj_data_info_sptr()->get_scanner_sptr()->get_scanner_geometry()=="Cylindrical")
+    {
+        out_2d_sptr.reset(new ProjDataInMemory(in_3d_sptr->get_exam_info_sptr(),
+                                               out_info_2d_sptr));
```

Comment:
as `out_info_2d_sptr` is set above to be the original, it isn't 2D. Therefore `SSRB` won't do anything.

Instead, `out_info_2d_sptr` needs to be set inside this `if` via a call to `SSRB` (as in the original code)

Reply:
yup. that's what it was before